Initium-Odp/src/com/universeprojects/miniup/server/OperationBase.java

```diff
@@ -35,6 +38,10 @@ public OperationBase(HttpServletRequest request, ODPDBAccess db)
 	{
 		this.db = db;
 		this.request = request;
+		
+		if(CommonChecks.checkCharacterIsZombie(db.getCurrentCharacter()))
+			if(!(this instanceof CommandSwitchCharacter))
```

Comment:
Why would you do this if you already have the check in the command?

Reply:
The command checks the target character, this checks the source character.